### PR TITLE
Homepage paths component - fixed buttons colors

### PR DESCRIPTION
**Pull request by Drorka**
resolving #101 

I changed the buttons setting to match Efi's directions.
So now the buttons work like this:
- Mobile view - displayed in color
- Desktop view - displayed in gray/white, and colored only when hovered.

Screenshot where I hovered over the middle button in desktop view:
image

### CODE REVIEW

File: components/Paths/PathItem.tsx [hunk 1]
```diff
@@ -19,9 +21,10 @@ export default function PathItem({ item }: PathProps) {
       ></div>
       <h4 className="capitalize self-center">{item.title}</h4>
       <div className="flex flex-col grow justify-between gap-10 min-h-[128px] items-center ">
-        <p className="text-base font-normal">{item.description}</p>
+        <p className="text-base font-normal font-inter">{item.description}</p>
         <a
-          className={`text-base font-normal px-4 py-2 body-bold ${item.btnColorDark} bg-lightText w-fit rounded-xl`}
+          className={`text-base font-normal font-inter px-4 py-2 body-bold md:bg-lightText md:dark:bg-gray-700 ${item.btnColorMobile}
```
Comment (adirkandel) [inline components/Paths/PathItem.tsx:26]: please use the `classNames` function to concatenate multiple classes

**Comment by adirkandel** [comment]
@Drorka left one comment, let me know when you finish.
You did a great job, thanks 👏

**Comment by Drorka** [comment]
@adirkandel thanks, I modified the code

**Comment by Drorka** [comment]
@adirkandel pinging you to check the fixes

File: components/Paths/PathItem.tsx [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
 }
 
 export default function PathItem({ item }: PathProps) {
+  const buttonClasses = classNames(
+    'text-base font-normal font-inter px-4 py-2 body-bold w-fit rounded-xl md:bg-lightText md:dark:bg-gray-700',
+    item.btnColor,
+    {
+      [`hover:${item.btnColor}`]: true,
+      [`hover:dark:${item.btnColor}`]: true,
+    }
```
Comment (adirkandel) [inline components/Paths/PathItem.tsx:21]: If you know that those classes will always get `true` then no need to use object just -> 
```suggestion
    `hover:${item.btnColor} hover:dark:${item.btnColor}`
```

**Review by adirkandel — APPROVED** [review]

**Comment by adirkandel** [comment]
@Drorka approved but with one comment to complete 🙏

**Comment by Drorka** [comment]
> @Drorka approved but with one comment to complete 🙏

Thanks for the review, I committed your suggestion

**Comment by Drorka** [comment]
@adirkandel I think this can be merged?

**Comment by adirkandel** [comment]
WELL DONE!!!!
@Drorka 👏 👏 👏 👏 👏

**Pull request merged**